### PR TITLE
Add election process description

### PR DESCRIPTION
**Pull request by ildikov**
This change adds a new folder to capture the election process and later on store email and other templates as needed.

The patch also adds some further information about CIVS and how to set up voting.

### CODE REVIEW

File: elections/process/README.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+- Create CIVS page
+  - Title the poll: <Year> <H1/H2> Kata Architecture Committee Election Poll
+  - Enable detailed ballot reporting
+  - Send to other officials to verify
+  - Check number of seats
+  - Check closing date
+
+
+When TSC Election begins:
```
Comment (stevenhorsman) [inline elections/process/README.md:108]:
```suggestion
When AC Election begins:
```

File: elections/process/README.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+- Upload rolls
+- CIVS has a maximum number of electorate emails you can upload at a time without
+  crashing, limit to 500 at a time
+- Send Kata AC election - Voting period started email, and announce the start
+  of the voting period on Slack
+
+A couple of days before the TSC Election ends:
```
Comment (stevenhorsman) [inline elections/process/README.md:116]:
```suggestion
A couple of days before the AC Election ends:
```

File: elections/process/README.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+During the AC Candidacy round:
+- Election officials review the nominations on GitHub
+- A couple of days before the candidacy submission ends send
+  reminders to the mailing list and Slack
+  - Mention this is the last call for candidate nomination
+  - Mention specifically the nomination deadline, and the
+    full timeline
+  - Example email from an earlier cycle:
+  	[October 2023 Kata Containers Architecture Committee elections candidate nomination reminder](https://lists.katacontainers.io/pipermail/kata-dev/2020-September/001512.html)
```
Comment (stevenhorsman) [inline elections/process/README.md:74]: Note this is from September 2020, not October 2023

**Comment by ildikov** [comment]
@stevenhorsman Thank you for the quick review! I fixed your comments.

**Review by stevenhorsman — APPROVED** [review]
Looks okay to me. Thanks!

**Review by lifupan — APPROVED** [review]

**Pull request merged**